crypto_app/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+CURRENCY_CHOICES = [
+        ("USD", "USD"),
+        ("PLN", "PLN"),
+        ("EUR", "Euro"),
+        ("GBP", "British Pound")
+]
+
+TRANSACTION_TYPE_CHOICES = [
+    ("BUY", "Purchase"),
+    ("SELL", "Sale")
+]
+
+
+class User(AbstractUser):
+    currency = models.CharField(max_length=3, choices=CURRENCY_CHOICES, default="PLN")
+
+    def __str__(self):
+        return self.username
+
+
+class Cryptocurrency(models.Model):
+    name = models.CharField(max_length=100)
+    symbol = models.CharField(max_length=10)
+    description = models.TextField()
+
+    def __str__(self):
+        return f"{self.name} ({self.symbol})"
+
+
+class MarketData(models.Model):
+    cryptocurrency = models.ForeignKey(Cryptocurrency, on_delete=models.CASCADE)
+    price = models.DecimalField(max_digits=20, decimal_places=8)
+    volume = models.DecimalField(max_digits=20, decimal_places=2)
+    date = models.DateTimeField(auto_now_add=True)
+    currency = models.CharField(max_length=3, choices=CURRENCY_CHOICES, default="PLN")
+
+    def __str__(self):
+        return f"{self.cryptocurrency.symbol} - {self.currency} - {self.price}"
+
+
+class Transaction(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    cryptocurrency = models.ForeignKey(Cryptocurrency, on_delete=models.CASCADE)
+    type = models.CharField(max_length=10, choices=TRANSACTION_TYPE_CHOICES)
+    amount = models.DecimalField(max_digits=20, decimal_places=8)
+    price = models.DecimalField(max_digits=20, decimal_places=2)
+    date = models.DateTimeField(auto_now_add=True)
```

Comment:
używasz `DateTimeField` a pole nazywa się `date` - nieintuicyjne

```suggestion
    created_at = models.DateTimeField(auto_now_add=True)
```